lib/otel/constants.js

```diff
@@ -107,11 +114,18 @@ module.exports = {
   ATTR_HTTP_URL: 'http.url',
 
   /**
-   * The http response status code
+   * The http response status code from undici library
```

Comment:
i wouldn't call out "from undici" or "from http".  `http.response.status_code` is per semantic conventions. The other is just being used but not spec compliant